test/conformance/source/environment.cpp

```diff
@@ -382,66 +382,44 @@ std::string KernelsEnvironment::getSupportedILPostfix(uint32_t device_index) {
         error = "failed to get backend from platform.";
         return {};
     }
-    if (backend == UR_PLATFORM_BACKEND_HIP) {
-        return ".bin";
-    }
 
-    auto device = instance->GetDevices()[device_index];
-    std::string IL_version;
-    if (uur::GetDeviceILVersion(device, IL_version)) {
-        error = "failed to get device IL version";
+    std::string target = "";
+    switch (backend) {
+    case UR_PLATFORM_BACKEND_OPENCL:
+    case UR_PLATFORM_BACKEND_LEVEL_ZERO:
+        return "spir64";
+    case UR_PLATFORM_BACKEND_CUDA:
+        return "nvptx64-nvidia-cuda";
+    case UR_PLATFORM_BACKEND_HIP:
+        return "amdgcn-amd-amdhsa";
+    case UR_PLATFORM_BACKEND_NATIVE_CPU:
+        error = "native_cpu doesn't support kernel tests yet";
         return {};
```

Comment:
Should we abort here to make it obvious something needs to change when support is being added?

Reply:
Aborting here would cause the entire test suite to die, which means we wouldn't be able to run test suites that have a mixture of "device code" and "non-device code" tests in them.

Any tests that use device code will check that they are loaded correctly (i.e. this function returns something non-empty), and if not they will fail quickly.

Reply:
Ah right. Makes sense.